### ISSUE TITLE
Please package Shutter Encoder

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#192179]
**Project description**

> A converter designed by video editors.
> The professional compression tool accessible to all.

It's the best GUI video encoder for linux i found. Right now we don't have a GUI video encoder packaged that supports ProRes.

It could be better with a native GUI (like GTK4) and not written in Java. [Ciano](https://robertsanseries.github.io/ciano/index.html) looks nice, but does not support ProRes ([yet](https://github.com/robertsanseries/ciano/issues/135)).

I tested the AppImage and it is exactly what i need. ffmpeg has too much mental load for a task that should be easy.

**Metadata**

* homepage URL: https://www.shutterencoder.com/
* source URL: https://github.com/paulpacifico/shutter-encoder
* license: gpl3+
* platforms: windows, linux, darwin

**Comment by literon36** [comment]
Bump! Definitely the best encoder/transcoder/decoder/converter out there! Havent found any good replacement on NixOS until now...

**Comment by Vaisakhkm2625** [comment]
yes.. i took looking for this... too bad haven't packaged it till now... i am too noob to package it myself....

**Comment by cbleslie** [comment]
Had a run at trying to package this myself. It has a few dependencies that don't have packages themselves. It will be more than trivial. Also, the way those deps are consumed for final packaging might be problematic. :(
